src/utils.test.js

```diff
@@ -2,7 +2,7 @@ import { asyncSetInterval, predictFileType, sortCompare } from './utils';
 
 
 describe('the utils helper', () => {
-  it('supports asyncSetInterval', (done) => {
+  it.skip('supports asyncSetInterval', (done) => {
```

Comment:
is there a TODO or ticket to fix this?

Reply:
Not yet, can create a ticket for this. @frickjack  Could you help me with this? Do we still need this test?